lib/addresses.ts

```diff
@@ -109,6 +125,8 @@ export async function resolveAddresses(
     case 2222:
       // Ganache
       return Deployments.ganache;
+    case 1337: // I have no idea why but it's consistent for me, someone else needs to confirm
```

Comment:
I get 31337 so I suggest we have both cases. The `buidler` should not be necessary anymore so let's have 1337 and 31337 both resolve to hardhat.